imports/ofx/account.go

```diff
@@ -0,0 +1,54 @@
+package ofx
+
+import (
+	"github.com/aclindsa/xml"
+	"gorm.io/gorm"
+)
+
+type Account struct {
+	gorm.Model
+	Currency     string
+	BankID       string
+	AccountID    string
+	Type         string
+	Transactions []Transaction
+}
+
+func (Account) TableName() string {
+	return tablePrefix + "accounts"
+}
+
+func (a *Account) UnmarshalXML(decoder *xml.Decoder, start xml.StartElement) error {
+	var data struct {
+		AccountInfo struct {
+			Currency string `xml:"CURDEF"`
+			Meta     struct {
+				BankID    string `xml:"BANKID"`
+				AccountID string `xml:"ACCTID"`
+				Type      string `xml:"ACCTTYPE"`
+			} `xml:"BANKACCTFROM"`
+			TransactionsInfo struct {
+				Transactions []Transaction `xml:"STMTTRN"`
+			} `xml:"BANKTRANLIST"`
```

Comment:
> * If the XML element contains a sub-element whose name matches
   the prefix of a tag formatted as "a" or "a>b>c", unmarshal
   will descend into the XML structure looking for elements with the
   given names, and will map the innermost elements to that struct
   field. A tag starting with ">" is equivalent to one starting
   with the field name followed by ">".

from https://golang.org/pkg/encoding/xml/#Unmarshal

---

I think we can do similar thing here
```suggestion
			Transactions []Transaction `xml:"BANKTRANLIST>STMTTRN"`
```

Don't know if it works, just a suggestion 😄